Asn1f2/Program.cs

```diff
@@ -419,15 +419,26 @@ public static int CheckSuccess(IEnumerable<string> args)
             }
             /*else */if (cmdArgs.HasArgument("c"))
             {
-                WriteTextFile(Path.Combine(outDir, "asn1crt.c"), Resource1.asn1crt);
-                WriteTextFile(Path.Combine(outDir, "asn1crt.h"), Resource1.asn1crt1);
-                WriteTextFile(Path.Combine(outDir, "acn.c"), Resource1.Acn);
-                WriteTextFile(Path.Combine(outDir, "real.c"), Resource1.real);
+                WriteTextFile(Path.Combine(outDir, "asn1crt_acn.h"), Resource1.asn1crt_acn_h);
+                WriteTextFile(Path.Combine(outDir, "asn1crt_ber.h"), Resource1.asn1crt_ber_h);
+                WriteTextFile(Path.Combine(outDir, "asn1crt_core.h"), Resource1.asn1crt_core_h);
+                WriteTextFile(Path.Combine(outDir, "asn1crt_real.h"), Resource1.asn1crt_real_h);
```

Comment:
wouldn't simple `real.h` be efficient? Users probably put code in some subdir of their projects, so includes will look nice (like `#include "asn1scc_generated/real.h").